packages/fast-form-generator-react/package.json

```diff
@@ -88,6 +88,10 @@
     "yargs": "^11.0.0"
   },
   "dependencies": {
-    "@microsoft/fast-jest-snapshots-react": "^1.0.0"
+    "@microsoft/fast-components-class-name-contracts-base": "^1.0.0",
+    "@microsoft/fast-glyphs-msft": "^1.0.0",
+    "@microsoft/fast-jest-snapshots-react": "^1.0.0",
+    "@microsoft/fast-jss-manager": "^1.0.0",
+    "@microsoft/fast-jss-utilities": "^1.0.0"
```

Comment:
you can remove fast-jest-snapshots-react from dependencies, if anything they should be in devDependencies, can also remove fast-glyphs-msft since we will be using just the pseudo elements for styling.

Reply:
Yep - whoops I preemptively added the glyphs package. Will fo on the other.